ParcelInvoiceMaker/ParcelInvoiceMaker/Feature/Invoice/Model/Parcel/ParcelReceiverInfo.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ParcelReceiverInfo.swift
+//  ParcelInvoiceMaker
+//
+//  Created by Choi Oliver on 3/4/24.
+//
+
+import Foundation
+
+protocol ValidatedString {
+    var value: String { get set }
+    
+    static func isValid(_ value: String) -> Bool
+    init(value: String) throws
+}
+
+struct ParcelReceiverName: ValidatedString {
+    var value: String
+    
+    static func isValid(_ value: String) -> Bool {
+        // 정규식 pattern. 한글, 영어, 숫자, 밑줄(_)만 있어야함
+        let pattern: String = "^[가-힣ㄱ-ㅎㅏ-ㅣa-zA-Z0-9_]$"
+        return pattern.range(of: value, options: .regularExpression) != nil
+    }
```

Comment:
검증 로직이 제대로 동작하지 않는것 같습니다 
```suggestion
   func isValid(_ value: String) -> Bool {
  // 정규식 pattern. 한글, 영어, 숫자, 밑줄(_)만 있어야함
  let pattern: String = "^[가-힣ㄱ-ㅎㅏ-ㅣa-zA-Z0-9_]$"
  return value.range(of: pattern, options: .regularExpression) != nil
}
```
pattern과 value가 바뀐것 같네요 !